app/controllers/commentVotes/voteComment.js

```diff
@@ -84,5 +84,5 @@ export const upsertCommentVote = async (query) => {
     },
   });
 
-  return { success: 'Vote created successfully!', commentVote };
+  return commentVote;
```

Comment:
Think this should be an object for it to work `return { commentVote }`

And remove the response.success assertions in the tests to make them pass 😄